core/src/tvu.rs

```diff
@@ -119,6 +120,25 @@ impl Default for TvuConfig {
     }
 }
 
+/// [`VoteClientOption`] enum represents the available client types for
+/// Vote communication:
+/// * [`ConnectionCacheClient`]: Uses a shared [`ConnectionCache`] to manage
+///   connections.
+/// * [`TpuClientNextClient`]: Relies on the `tpu-client-next` crate.
+pub enum VoteClientOption<'a> {
```

Comment:
Can we not reuse a similar enum that exists for transaction clients?

Reply:
There is ClientOption defined in rpc and also a ForwardingClientOption for awarding in core with some difference on the multihoming arguments. I will see what we can do